packages/angular/src/flyout-menu/flyout-menu-pane.component.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Optional,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  AnimationFrameService,
+  CloseMeta,
+  closestAttr,
+  Dialog,
+  ElementService,
+  position,
+} from 'carbon-components-angular';
+import { I18n } from 'carbon-components-angular/i18n';
+
+/**
+ * The Filter menu component encapsulates the OverFlowMenu directive, and the flyout iconography
+ * into one convienent component
+ *
+ * [See demo](../../?path=/story/components-flyout-menu--basic)
+ *
+ * html:
+ * ```
+ * <ai-flyout-menu-pane>
+ *	options
+ * </ai-flyout-menu-pane>
+ * ```
+ *
+ * <example-url>../../iframe.html?id=components-flyout-menu--basic</example-url>
```

Comment:
I don't think `example-url` does anything at the moment, with the switch to tsdoc ... I bet we could make a webcomponent that renders the example though 🤔